### PR TITLE
add home page and about page for desktop and mobile user

### PR DESCRIPTION
**Pull request by milen-ale**
In this capstone project, I build a website for Microverse by following the design guidelines including Color, Typograpghy, and Layout from the original design on Behance by Cindy Shin. During this project, I designed two different pages

Home Page
About Page
I designed these two pages for both mobile and desktop users.

- here is a link to a short description of my capstone project https://www.loom.com/share/5a89701801e8417282cde4db3befb2d8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1 @@
...
-   ``` 
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
-
-2. Copy [.eslintrc.json](./.eslintrc.json) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run `npx eslint .` on the root of your directory of your project.
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+# capestone-project-
```
Comment (uwadonat) [inline README.md]: - [ ]  Please update your readme file. You can follow Microverse [template](https://github.com/microverseinc/readme-template/blob/master/README.md).

- [ ] Kindly add link for a loom video showing project in desktop and mobile version. Again highlight peace of code you are proud of.
- [ ] You should use of GitHub Pages to deploy the web pages and add link in your readme file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                  <li><a href="#" class="link2 link">Sponsors</a></li>
+                  <li><a href="#" class="link3 link">Donate</a></li>
+                  <li><a href="#" class="link4 link">Contact us</a></li>
+                </ul>
+              </div>
+            </nav>
+          </div>
+
+          <img class="humburger" src="./img/hamburger-icon.svg" alt="" />
+
```
Comment (uwadonat) [inline index.html]: - [ ] Make sure all images contain alt attributes. Example: `alt="my-image"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        <!-- <div desktop-header>
+        <ul class="main-nav-list ">
+          <li><a href="index.html" class=" link0  link">Home</a></li>
+          <li><a href="about.html" class="link1 link">About us</a></li>
+          <li><a href="#" class="link2 link">Sponsors</a></li>
+          <li><a href="#" class="link3  link">Donate</a></li>
+          <li><a href="#" class="link4  link">Contact us</a></li>
+        </ul> 
+      </div>-->
+        <div>
```
Comment (uwadonat) [inline index.html]: - [ ] Please remove all commented lines to make your project more clean

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @milen-ale, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your design looks nice ✔️
- No linter errors ✔️
- You are correctly using github flow ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,101 +1,53 @@
...
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+## Live Demo
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+You can checkout the live version [here](https://milen-ale.github.io/capstone/)
```
Comment (medaminedev66) [inline README.md]: - [ ] The live demo link is broken. Kindly fix this so that your project is complete.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @milen-ale 😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  The video presentation was really good.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi team 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**